include/units.h

```diff
@@ -2041,7 +2191,7 @@ namespace units
 		inline constexpr operator Ty() const noexcept 
 		{ 
 			// this conversion also resolves any PI exponents, by converting from a non-zero PI ratio to a zero-pi ratio.
-			return static_cast<Ty>(units::convert<UnitType, units::unit_conversion<std::ratio<1>, units::dimension::dimensionless>>((*this)()));
+			return units::convert<units::unit<units::unit_conversion<std::ratio<1>, units::dimension::dimensionless>, Ty, NonLinearScale>>(*this)();

```

Comment:
love this.

Reply:
It took me a while to correct this, which was the last use of the non-type safe `convert`.